Added docs for enabling opensearch (#100)
rapidez/docs: src/4.x/configuration.md

SCOUT_PREFIX=
```

## Opensearch

> To communicate with Opensearch we also utilise [rapidez/laravel-scout-elasticsearch](https://github.com/rapidez/laravel-scout-elasticsearch).
> 
> Configuration is the same as [Elasticsearch](#elasticsearch).
> 
> But you add
> 
> ```dotenv
> SCOUT_SEARCH_BACKEND=opensearch
> ```
> 
> to your .env

Reviewer: ````suggestion
For OpenSearch the same applies but you need to add this in your `.env`:

```dotenv
SCOUT_SEARCH_BACKEND=opensearch
```
````
